stt_configs:
  compute_type: "float32"
  spinner: False
  model: "distil-small.en"
  silero_sensitivity: 0.6
  device: "cpu"
  webrtc_sensitivity: 1
  post_speech_silence_duration: 1.2
  min_length_of_recording: 3.0
  min_gap_between_recordings: 0
  enable_realtime_transcription: False
  silero_deactivity_detection: True
  initial_prompt: "Goias Robocup 2006"

stt_mic_timeout: 10
